### PR TITLE
Update documentation of get_duration()

### PR DESCRIPTION
**Pull request by nehz**
Updated docs for #726

---
This change is [Reviewable](https://reviewable.io/reviews/librosa/librosa/727)

### CODE REVIEW

File: librosa/core/audio.py [hunk 1]
```diff
@@ -332,6 +332,8 @@ def get_duration(y=None, sr=22050, S=None, n_fft=2048, hop_length=512,
     S : np.ndarray [shape=(d, t)] or None
         STFT matrix, or any STFT-derived matrix (e.g., chromagram
         or mel spectrogram).
+        The duration returned using this option will only be an
+        estimated value, reconstructed using the integer frames
```
Comment (bmcfee) [inline librosa/core/audio.py]: "reconstructed using the integer frames" might not be totally clear to a reader.

Suggested rephrase: "Durations calculated from spectrogram inputs are only accurate up to the frame resolution.  If high precision is required, it is better to use the audio time series directly."

... or something to that effect.

**Comment by bmcfee** [comment]
Reviewed 1 of 1 files at r2.
Review status: all files reviewed, 1 unresolved discussion (waiting on @bmcfee)

---



*Comments from [Reviewable](https://reviewable.io/reviews/librosa/librosa/727)*

**Comment by bmcfee** [comment]
:lgtm:

---

Review status: all files reviewed, 1 unresolved discussion (waiting on @bmcfee)

---



*Comments from [Reviewable](https://reviewable.io/reviews/librosa/librosa/727#-:-LEyqxujAQgPxllSR0Dx:bnfp4nl)*

**Comment by bmcfee** [comment]
Thanks again for the contribution!  Everything looks good to me.  Do you have anything else to add, or shall I merge?

**Comment by nehz** [comment]
No worries, all looks good

**Pull request merged**